src/config/dynamic-theme-fixes.config

```diff
@@ -13751,6 +13751,18 @@ INVERT
 
 ================================
 
+wiki.facepunch.com
+
+CSS
+body {
```

Comment:
It doesn't seem `body` has any background image?

Reply:
wiki.facepunch.com/gmod have repeating background image (3x3) in <div class="body">. Without dark reader, this image hard to see. But when dark reader inverts this image, it is looks weird. https://imgur.com/QJvG365

Later I'm will send you devtools with css properties.

Reply:
\<body> doesn't have any background (color or image). But dark reader adds background color to it